components/nautobot/values.yaml

```diff
@@ -47,6 +49,7 @@ celery:
   replicaCount: 1
   extraEnvVarsSecret:
     - nautobot-django
```

Comment:
```suggestion
    - nautobot-django
```

This can be removed as well since you're not stuffing extra bits into there.

Reply:
If we remove it, then how does the secret key make it in to nautobot? I don't see any mention of "nautobot-django" in https://github.com/nautobot/helm-charts so it doesn't look like nautobot does that for us today.

Reply:
That's just the celery spot. AFAIK, celery doesn't need it?

Reply:
we can keep it for now then.